src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/ProjectServices/VsProjectBuildProperties.cs

```diff
@@ -1,30 +1,35 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
 using System.Threading.Tasks;
 using Microsoft;
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.Shell.Interop;
 using NuGet.ProjectManagement;
 
 namespace NuGet.PackageManagement.VisualStudio
 {
     /// <summary>
-    /// Contains the information specific to a Visual Basic or C# project.
+    /// Contains the information specific to a Visual Basic or C# project or NetCore project.
     /// </summary>
-    internal class VsCoreProjectBuildProperties
+    internal class VsManagedLanguagesProjectBuildProperties
```

Comment:
Doesn't this need to be VsProjectBuildProperties? 
It's different from the file name.